index.html

```diff
@@ -1 +1,13 @@
-<h1>Hello World</h1>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
+</head>
+<body>
+  <h1>TITLE</h1>
```

Comment:
Don't use all capitalize letters here. Please adjust

Reply:
changed all letters to lowercase